### PR TITLE
Fix misc. typos and whitespace

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

**Comment by luzpaz** [comment]
@ChrisVeigl please review

**Comment by ChrisVeigl** [comment]
thanks! - sorry this took ages ;)

File: src/ob_eeg.cpp [hunk 1]
```diff
@@ -50,7 +50,7 @@
   process_packets:  calls the worker-functions of all existing objects.
   parse_byte_P2: parses a ModEEG-P2 datastream and 
 			stores channel values into the packetstructure
-  parse_byte_P21: parses a P21 datastream (by Jarek Foltynski, modified by Reiner M�nch)
+  parse_byte_P21: parses a P21 datastream (by Jarek Foltynski, modified by Reiner M�nch)
```
Comment (luzpaz) [inline src/ob_eeg.cpp:53]: Just realized the editor I used auto-corrected this incorrectly.

File: src/ob_eeg.cpp [hunk 2]
```diff
@@ -277,7 +277,7 @@ void parse_byte_P3(unsigned char actbyte)
 
   16 bit Raw binary Data Format:
    
-	 1 channel, �given in two byte (low byte first)
+	 1 channel, �given in two byte (low byte first)
```
Comment (luzpaz) [inline src/ob_eeg.cpp:280]: same here

File: src/ob_eeg.cpp [hunk 3]
```diff
@@ -320,7 +320,7 @@ void parse_byte_raw_8bit(unsigned char actbyte)
 
 /*********************************************************************************
 
-  Modular EEG Protocol P21 with add-on ( by Jarek Foltynski and Reiner M�nch)
+  Modular EEG Protocol P21 with add-on ( by Jarek Foltynski and Reiner M�nch)
```
Comment (luzpaz) [inline src/ob_eeg.cpp:323]: and here

**Comment by ChrisVeigl** [comment]
ok - no problem - i'll fix those in the master branch

**Comment by luzpaz** [comment]
Sorry about that!

**Comment by ChrisVeigl** [comment]
no prob! - thank you very much for your contribution!

**Pull request merged**